ompi/mca/fs/lustre/fs_lustre_component.c

```diff
@@ -93,6 +95,15 @@ lustre_register(void)
                                            MCA_BASE_VAR_TYPE_INT, NULL, 0, 0,
                                            OPAL_INFO_LVL_9,
                                            MCA_BASE_VAR_SCOPE_READONLY, &mca_fs_lustre_stripe_width);
+    mca_fs_lustre_lock_algorithm = 0;
+    (void) mca_base_component_var_register(&mca_fs_lustre_component.fsm_version,
+                                           "lock_algorithm", "Locking algorithm used by the fs ufs component. "
+                                           " 0: auto (default), 1: skip locking, 2: always lock entire file, "
+                                           "3: lock only specific ranges",
```

Comment:
If this MCA parameter only accepts specific values, is there any reason not to use the enum MCA type? (such that the MCA base will take care of ensuring that only allowable values are set)

Reply:
@jsquyres you are technically correct that we could enum MCA types, this is more a feature of what I am used to doing vs. what is possible. That being said, the same pr has already been merged to 5.0.x as well, do we really want to go back and change that as well? We could still fix it in main branch for future releases.